packages/react-core/src/components/Label/examples/Label.md

```diff
@@ -30,7 +30,9 @@ import './Label.css';
 
 ```
 
-### Label with router link
+### Label with custom render
+
+Labels may be passed a custom renderer to display customized content or for use with router components.
 
 ```ts file="LabelRouterLink.tsx"
```

Comment:
Related to my PR https://github.com/patternfly/patternfly-react/pull/11545, but I had removed these examples from components and instead added a separate section for using router links. Maybe we keep these examples (renamed to "Custom render/component", depending on the prop name) as well as a separate section related to router links?

Reply:
Yeah I think (at least in this case) we can keep the example as a custom render example, since the prop can be used for more than just rendering router links.

Reply:
Sounds good, I'll make updates to my PR then to follow (I won't add back this example, though, we can deal with rebasing depending whose PR gets in first).

Could we just update the name of this example file to match the new purpose, `LabelCustomRender`?

Reply:
Updated the file name and also cleaned up the example after seeing how the router link example will look in your PR (the inline comments aren't needed anymore).